### ISSUE TITLE
plugin don't load / Outdated API 3.0.0 / Call to undefined method KitUI/Main" at line 43

### ISSUE THREAD

**Issue by teracube** [Infernus101/KitUI#86]
## KitUI Issue
- [X] This issue isn't duplicated - you can check if it is by using the search bar located at the top left hand corner and select "Issues" on the left.
- [X] This issue includes appropriate markdown for sections - e.g. code blocks for crash dumps.
- [X] This issue is understandable in English.

## Issue description
...
plugin don't load in API 3.0.0

## Reproducing the issue
1. ...install https://jenkins.pmmp.io/job/PocketMine-MP/1202/artifact/PocketMine-MP.phar
2. ...install https://poggit.pmmp.io/r/27126/KitUI_dev-65.phar
in plugin.yml i modify API 3.0.0

## Client information
PocketMine-MP Version: (insert commit hash here)
Kit-UI Version: (insert commit hash or version from plugin.yml here)
MCPE Version: (insert game version here)
Plugin Version: (insert commit hash here)
PHP version: 7.2

## Optional information
```

2018-06-20 [01:20:08] [Server thread/INFO]: Enabling KitUI v1.2.2
2018-06-20 [01:20:08] [Server thread/NOTICE]: [KitUI] Enabled! - By Infernus101
2018-06-20 [01:20:08] [Server thread/CRITICAL]: Error: "Call to undefined method pocketmine\Server::getScheduler()" (EXCEPTION) in "KitUI-master/src/Infernus101/KitUI/Main" at line 43
2018-06-20 [01:20:08] [Server thread/DEBUG]: #0 src/pocketmine/plugin/PluginBase(102): Infernus101\KitUI\Main->onEnable()
2018-06-20 [01:20:08] [Server thread/DEBUG]: #1 src/pocketmine/plugin/PluginManager(618): pocketmine\plugin\PluginBase->setEnabled(boolean 1)
2018-06-20 [01:20:08] [Server thread/DEBUG]: #2 src/pocketmine/Server(1928): pocketmine\plugin\PluginManager->enablePlugin(Infernus101\KitUI\Main object)
2018-06-20 [01:20:08] [Server thread/DEBUG]: #3 src/pocketmine/Server(1914): pocketmine\Server->enablePlugin(Infernus101\KitUI\Main object)
2018-06-20 [01:20:08] [Server thread/DEBUG]: #4 src/pocketmine/Server(1708): pocketmine\Server->enablePlugins(integer 1)
2018-06-20 [01:20:08] [Server thread/DEBUG]: #5 src/pocketmine/PocketMine(237): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/gsp_1485/g4.hmserv.eu-19140/, string /home/gsp_1485/g4.hmserv.eu-19140/plugins/)
2018-06-20 [01:20:08] [Server thread/DEBUG]: #6 /home/gsp_1485/g4.hmserv.eu-19140/PocketMine-MP.phar(1): require(string phar:///home/gsp_1485/g4.hmserv.eu-19140/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
2018-06-20 [01:20:08] [Server thread/INFO]: Disabling KitUI v1.2.2

```
Replace this text with console / error logs or crash dumps. (Do not remove the triple backticks above and below this.)
```

<!-- Config - paste kits.yml if required -->
## Config
...

**Comment by TDMidkwhatisxd** [comment]
i think is dead, time to find another plugin

**Comment by JaxkDev** [comment]
The scheduler has changed to each plugin in the 3.0.0 update no Async functions would work on older versions.

**Comment by Infernus101** [comment]
Fixed in v1.3.0
